src/components/portrait-view.tsx

```diff
@@ -67,68 +68,34 @@ export const PortraitViewRow = (props: PortraitViewRowProps) => {
 
 export const PortraitView = (props: ITableProps) => {
   const {collectionClasses, selectedDataSet, collections, getValueLength} = props;
+  const tableRef = useRef<HTMLTableElement | null>(null);
+  const tableScrollTop = useTableScrollTop(tableRef);
+  const [resizeCounter, setResizeCounter] = useState(0);
+
   const thresh = useMemo(() => {
     const t: number[] = [];
     for (let i = 0; i <= 100; i++) {
-      t.push(i / 100);
+      t.push(i/100);
     }
     return t;
-  },[]);
+  }, []);
 
-  const [scrolling, setScrolling] = useState(false);
-  const [scrollTop, setScrollTop] = useState(0);
 
   useEffect(() => {
```

Comment:
Do we still need this or can it be replaced by a window resize event handler?

Reply:
@dougmartin I'm assuming you are commenting on the IntersectionObserver and not the two scrolling states. 
We are using the IntersectionObserver for the window resize. Not sure if it is better or worse than using a ResizeObserver.

Reply:
I think the window resize event is "lighter" but at this point lets keep it as is and put this to bed :)